charts/load-test-grpc/values.yaml

```diff
@@ -41,3 +41,21 @@
 ###   grpc/latency/mean: 50
 ###   grpc/latency/p99: 200
 # SLOs: null
+SLOs: null
+
+### readinessDefinitions   A map of Kubernetes resource kinds to group/version/resource and condition
+### group/version/kind and namespace/name can be used to uniquely identify a resource in a cluster
+### condition is a status condition that indicates the resource is ready for use
+readinessDefinitions:
```

Comment:
The readiness definitions are essentially constants, as opposed to other values, like deployment name.
***
Could we please move these definitions under `iter8lib` chart's `values.yaml`, especially, under its global values?

```yaml
# iter8lib/values.yaml
global:
  readinessDefinitions:
    deploy: ...
```

It can then be set/accessed as `iter8lib.global.values.resource` in every other chart that embeds it.

Reply:
done